src/main/resources/locales/en/opa-public-list.json

```diff
@@ -0,0 +1,24 @@
+{
+    "title": "OPA Public List",
+    "heading": "Online Plea and Allocation Cases for",
+    "listDetail": "List containing CASES_COUNT case(s) generated on",
+    "publishedAt": "at",
+    "restrictionInformationHeading": "Restrictions on publishing or writing about these cases",
+    "restrictionInformationP1": "You must check if any reporting restrictions apply before publishing details on any of the cases listed here either in writing, in a broadcast or by internet, including social media.",
+    "restrictionInformationBoldText": "You'll be in contempt of court if you publish any information which is protected by a reporting restriction. You could get a fine, prison sentence or both.",
+    "restrictionInformationP2": "Specific restrictions ordered by the court will be mentioned on the cases listed here.",
+    "restrictionInformationP3": "However, restrictions are not always listed. Some apply automatically. For example, anonymity given to the victims of certain sexual offences.",
+    "restrictionInformationP4": "To find out which reporting restrictions apply on a specific case, contact:",
+    "restrictionBulletPoint1": "the court directly",
+    "restrictionBulletPoint2": "HM Courts and Tribunals Service on 0330 808 4407",
+    "restrictionInformationP5": "You can also read the",
```

Comment:
You can remove the following lines. It has been confirmed with Chris Carter we dont need to display the reporting restriction link anymore because the link does not exists

restrictionInformationP5
restrictionLink